runtime_config.go

```diff
@@ -1,9 +1,11 @@
 package specs
 
 type RuntimeSpec struct {
-	// Mounts profile configuration for adding mounts to the container's filesystem.
-	Mounts []Mount `json:"mounts"`
-	// Hooks are the commands run at various lifecycle events of the container.
+	// Mounts is a mapping of names to mount configuration.
+	// Which mounts will be mounted and where should be chosen with MountPoints
+	// in Spec.
+	Mounts map[string]Mount `json:"mounts"`
+	// Hooks are the commands run at various lifecycle events of the containers.
```

Comment:
On Wed, Sep 02, 2015 at 01:12:41PM -0700, Alexander Morozov wrote:
> > -	Mounts []Mount `json:"mounts"`
> > -	// Hooks are the commands run at various lifecycle events of the container.
> > …
> > +	Mounts map[string]Mount `json:"mounts"`
> > +	// Hooks are the commands run at various lifecycle events of the containers.
> 
> I don't see what changed in this comment.

The post-Mounts comment should be about *hooks*, and there's only one
*container* for lifecycle events.

> > +	// Mounts is a mapping of names to mount configuration.
> > +	// Which mounts will be mounted and where should be chosen with MountPoints
> > +	// in Spec.

The pre-Mounts comment^, on the other hand should be talking about
“mount configurations”, not “mount configuration”.